### PR TITLE
4826: Add dashboard page with header and footer

### PR DESCRIPTION
**Pull request by kat1906**
Story: [AB#4826](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/4826)
Task: [AB#6752](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6752)

### CODE REVIEW

File: app/config.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ module.exports = {
   // The base uri the app is running on to pass to identity service for redirection
   appBaseUri: process.env.APP_BASE_URI || 'http://localhost:3006',
 
+  // BLOBSTORE_HOST
+  blobstoreHost: process.env.BLOBSTORE_HOST || 'https://gpitfuturesdevsa.blob.core.windows.net',
```
Comment (alibombaye) [inline app/config.js:21]: why do we need this?

Comment (alibombaye) [reply app/config.js:21]: ah ok ignore me I can see why

File: app/config.js [hunk 2]
```diff
@@ -42,7 +45,7 @@ module.exports = {
   redisUrl: process.env.REDIS_URL || 'localhost',
 
   // The port redis is running
-  redisPort: process.env.REDIS_PORT || 6379,
+  redisPort: process.env.REDIS_PORT || 6380,
```
Comment (alibombaye) [inline app/config.js:48]: oh yeaahhh!!!! makes me wonder how I'm getting it to work for me locally via the cluster

Comment (kat1906) [reply app/config.js:48]: I think the cluster one is 6379 and local is 6380 so I will change it back

Comment (alibombaye) [reply app/config.js]: yeah that's right. got confused

**Pull request merged**